src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -150,6 +150,9 @@
     "span.group": PREFIX + 252,
     "transaction.method": PREFIX + 253,
     "span.category": PREFIX + 254,
+    "span.contribution": PREFIX + 255,
+    "span.main_thread": PREFIX + 256,
+    "device.class": PREFIX + 257,
```

Comment:
I was not able to check these strings against `sentry_stringindexer` and `sentry_perfstringindexer` yet. Blocking merge until we are sure that these strings are not already in use.

Reply:
Now verified.